libs/api/domains/social-insurance/src/lib/models/pensionCalculation.model.ts

```diff
@@ -0,0 +1,32 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+import { CacheField } from '@island.is/nest/graphql'
+
+@ObjectType('SocialInsurancePensionCalculationResponseItem')
+class PensionCalculationItem {
+  @Field(() => String, { nullable: true })
+  name?: string | null
```

Comment:
Isn't it unnecessary to have it be both `null` and `nullable`?

Reply:
I want to keep it though 
![image](https://github.com/island-is/island.is/assets/43557895/65d50c26-35a1-43c0-9839-e37fa9598a6d)